### PR TITLE
Tailwind future

### PR DESCRIPTION
**Pull request by RNR1**
Remove the Tailwind deprecation warning raises on gatsby build

### CODE REVIEW

File: site/src/utils/url.js [hunk 1]
```diff
@@ -6,9 +6,10 @@ export function parseTwitterHandle(url) {
   let handle
 
   try {
-    handle = url.match(TwitterRegexp)[1]
+    const [match] = url.match(TwitterRegexp)[1]
```
Comment (bloudermilk) [inline site/src/utils/url.js:9]: Is this functionally equivalent? It looks like we're double-destructuring now

File: site/tailwind.config.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  future: {
+    removeDeprecatedGapUtilities: true,
```
Comment (bloudermilk) [inline site/tailwind.config.js:3]: Nice, was just reading the page about that warning yesterday. Thanks!

File: site/src/utils/url.js (hunk 1 above)
Comment (RNR1) [reply site/src/utils/url.js]: I made it clearer now :)

File: site/src/utils/url.js [hunk 3]
```diff
@@ -3,13 +3,10 @@ const TwitterRegexp = /twitter\.com\/@?(\w+)/i
 export function parseTwitterHandle(url) {
   if (!url) return null
 
-  let handle
-
   try {
-    handle = url.match(TwitterRegexp)[1]
+    const [, handle] = url.match(TwitterRegexp)
```
Comment (RNR1) [inline site/src/utils/url.js:7]: my bad, now it works as expected :)

**Pull request merged**